index.bs

```diff
@@ -501,15 +501,21 @@ Automation {#automation}
 
 This section extends [[GENERIC-SENSOR#automation]] by providing [=Orientation Sensor=]-specific virtual sensor metadata.
 
-<div algorithm>
-The <dfn>orientation reading parsing algorithm</dfn>, given a JSON {{Object}} |parameters|, must return the result of invoking [=parse quaternion reading=] with |parameters|.
-</div>
+<h3 dfn> Absolute Orientation Sensor automation</h3>
```

Comment:
Because chapters content looks very similar, I thought that subheadings will make the text and Automation-chapter clearer. At lest to me.

If I remove dnf from heading I get "WARNING: The heading 'Absolute Orientation Sensor automation' needs a manually-specified ID." warning.

I would like to use 
subheading_text
------------------
style as after uploading the pull request noticed that this style was also used in document. But in that case "{#subheading-ID}" needs to be defined.

Reply:
> If I remove dnf from heading I get "WARNING: The heading 'Absolute Orientation Sensor automation' needs a manually-specified ID." warning.

Bikeshed needs _something_ for the id attribute of the `<h3>` tag that is generated, but it does not need to be a `dfn`. After all, a definition has a specific meaning and is normally something you refer to in other parts of the text, which is not the case here. You can just do the same thing as the text that got removed and use an "id" directly:

    <h3 id="absolute-orientation-sensor-automation">Absolute Orientation Sensor Automation</h3>

(don't forget to remove the extra white space before "Absolute" that you have there.

Reply:
Fixed in https://github.com/w3c/orientation-sensor/pull/78/commits/f1680ff9971efc4421e86956ae02fb39c84d8715